tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -283,7 +283,15 @@ def run(self):
                     _cluster = _cluster.load()
                     _cluster.status = ""
                     _cluster.last_sync = str(tendrl_now())
-                    _cluster.is_managed = "yes"
+                    # Mark the first sync done flag
+                    _cnc = NS.tendrl.objects.ClusterNodeContext(
+                        node_id=NS.node_context.node_id
+                    ).load()
+                    if _cnc.first_sync_done is None or \
```

Comment:
simplify this   (None and "" are the same)

if _cnc.first_sync_done in [None, "no"] : _cnc.first_sync_done = "yes"

Reply:
done